packages/sdk-js/src/handlers/activity.ts

```diff
@@ -1,24 +1,43 @@
 import { AxiosInstance } from 'axios';
 import { ItemsHandler } from './items';
-import { Query, PrimaryKey, Item, Response } from '../types';
+import { Query, PrimaryKey, Item, ItemsResponse } from '../types';
 
+export type ActivityItem = {
+	action: string;
+	ip: string;
+	item: PrimaryKey;
+	user_agent: string;
+	timestamp: string;
+	id: number;
+	user: string;
+	comment: string | null;
+	collection: string;
+	revisions: [number] | null;
```

Comment:
When I tested the api, `revisions` always return either null or array with single number inside. Maybe I don't have enough activity data to get different results